### DISCUSSION TITLE
color scheme for mapping quality

### DISCUSSION THREAD

**Discussion by yeeus** [GMOD/jbrowse-components#4244] [Q&A] [answered]
Hello, I'm using jBrowse2 to visualize my alignment track. It's useful for me to show the mapping quality by the seting "color scheme". However, I want to know the mapping quality score range each color representing. How can I get this information? 
Best wishes!

**Comment by cmdcolin** [comment, marked as answer]
hi there, it is a bit tricky because we don't properly have legends on our tracks. essentially the color is a HSL map


the function that does it is

```
export function colorByMappingQuality(feature: Feature) {
  return `hsl(${feature.get('score')},50%,50%)`
}
```

where score is the "MAPQ" value, and the value in practice ranges from about 0-100 (high quality illumina reads that i see have mapq of about 70)

so an example legend showing 0-100 mapq is here

![image](https://github.com/GMOD/jbrowse-components/assets/6511937/6abb0167-c288-4a52-9a1a-82250431a6d4)


hope that helps, we will keep the concept of adding legends on the radar so that it is not so unclear

**Reply by cmdcolin** [reply]
note that the per-base quality color scale is a little different, but the above one is the MAPQ
